lib/matplotlib/text.py

```diff
@@ -761,9 +764,7 @@ def draw(self, renderer):
             posy = float(textobj.convert_yunits(textobj._y))
             posx, posy = trans.transform_point((posx, posy))
             if not np.isfinite(posx) or not np.isfinite(posy):
-                verbose.report("x and y are not finite values for text "
-                              "string '{}'.  Not rendering "
-                              "text.".format(self.get_text()), 'helpful')
+                _log.warning("posx and posy should be finite values")
```

Comment:
I suspect this change in the message content has already been discussed and approved: correct?

Reply:
Yes - umm, I forget where, but I pushed this change...